pkg/didcomm/protocol/introduce/service.go

```diff
@@ -290,8 +242,8 @@ func (s *Service) InvitationReceived(msg service.StateMsg) error {
 	_, err := s.HandleInbound(service.NewDIDCommMsgMap(&model.Ack{
 		Type:   AckMsgType,
 		ID:     uuid.New().String(),
-		Thread: &decorator.Thread{ID: h.Thread.PID},
-	}), "", "")
+		Thread: &decorator.Thread{ID: msg.Msg.ParentThreadID()},
+	}), "internal", "internal")
```

Comment:
internal?

Reply:
We do not need it. Instead of empty values, I have added `internal` identifier. It may help in the case of debugging. Instead of looking for why values are empty you will receive it `internal`.